RecoTracker/TrackProducer/plugins/GsfVertexConstraintProducer.cc

```diff
@@ -18,14 +18,13 @@
 // class decleration
 //
 
-class GsfVertexConstraintProducer: public edm::EDProducer {
+class GsfVertexConstraintProducer: public edm::stream::EDProducer<> {
```

Comment:
Looks like this one could be global.

Reply:
I'm sorry but I don't know what you mean

Reply:
The EDProducer type could be global
https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkGlobalModuleInterface
Global is preferred when possible because then framework creates only one instance of the module (instead of the number of streams) leading to smaller use of memory.